src/geometries/ParametricGeometry.js

```diff
@@ -72,6 +77,16 @@ function ParametricBufferGeometry( func, slices, stacks ) {
 			var p = func( u, v );
 			vertices.push( p.x, p.y, p.z );
 
+			// approximate tangent plane vectors via central difference
+
+			pu.subVectors( func( u + EPS, v ), func( u - EPS, v ) );
+			pv.subVectors( func( u, v + EPS ), func( u, v - EPS ) );
```

Comment:
Should both `Vector3`'s returned by `func` be `.clone`ed here in case both objects are actually the same? Or is there a better way to do this?